kubernetes/build.gradle

```diff
@@ -38,9 +40,13 @@ task createRDockerfile(type: Dockerfile) {
 
 task createPythonDockerfile(type: Dockerfile) {
   destFile = outputFilePython
-  from "spark-py:${sparkVersion}"
+  arg "spark_version"
+  from 'spark-py:$spark_version'
+  arg "spark_version"
   copyFile("h2o_pysparkling_${sparkMajorVersion}-${version}.zip", "/opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip")
-  runCommand "pip install /opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip"
+  runCommand 'echo "pyspark==$spark_version" > constraints'
```

Comment:
We should run all the last 3 `runCommand` inside one `runCommand`. This way docker greates 3 layers, one with contastrains file, one with the packages and one with the removed constrains file -> bigger size.

If we do all the changes in one scope, docker won't save the intermediate files

Reply:
ok, thanks!